includes/StandardRenderer.h

```diff
@@ -41,10 +41,13 @@ class StandardRenderer : public IRenderComponent {
 
     void render();
     void renderShadow(std::shared_ptr<ShaderProgram> &shaderProgram);
+    void renderEmissive(std::shared_ptr<ShaderProgram> &shaderProgram);
```

Comment:
I don't know how doxygen works when overriding stuff. Is the documentation copied from the base class?